src/output/render/groups.rs

```diff
@@ -53,20 +56,17 @@ impl Render for Option<f::Group> {
             UserFormat::Numeric => group.gid().to_string(),
         };
 
-        group_name = match group_format {
-            GroupFormat::Smart => {
-                if let Some(current_user) = users.get_user_by_uid(current_uid) {
-                    if current_user.name() == group.name() {
-                        ":".to_string()
-                    } else {
-                        group_name
+        if let GroupFormat::Smart = group_format {
+            if let Some(file_uid) = file_user {
+                if let Some(file_user) = users.get_user_by_uid(file_uid.0) {
+                    let file_user_name = file_user.name().to_string_lossy().to_string();
+                    let group_name_text = group.name().to_string_lossy().to_string();
+                    if file_user_name == group_name_text {
```

Comment:
Seems to me like you can just do (at least rust-analyzer is OK with this code):

```suggestion
                    if file_user.name().to_string_lossy() == group.name().to_string_lossy() {
```

Reply:
Thanks! It is much better